tex/iii.1/_0079.tex

```diff
@@ -1,3 +1,4 @@
+\baselineskip
```

Comment:
This what causes strange line break for all pages below it. What was the intention?